components/tours/onboarding_explore_tools_tour/explore_tools_tour_manager.tsx

```diff
@@ -1,225 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Is this removal intended?

Reply:
yes, no longer needed. It was a leftover after the refactoring made to the tourtips.